support-models/src/main/scala/com/gu/support/zuora/api/response/Responses.scala

```diff
@@ -80,14 +81,15 @@ case class GetAccountResponse(success: Boolean, basicInfo: BasicInfo) extends Zu
 
 //this response cannot extend ZuoraResponse because this endpoint doesn't return a 'success' boolean field
 case class GetObjectAccountResponse(
-   IdentityId__c: Option[String],
-   sfContactId__c: Option[String],
-   CrmId: Option[String],
-   DefaultPaymentMethodId: Option[String],
-   AutoPay: Boolean,
-   Balance: Double,
-   Currency: String
-  )
+  IdentityId__c: Option[String],
+  sfContactId__c: Option[String],
+  CrmId: Option[String],
+  DefaultPaymentMethodId: Option[String],
+  AutoPay: Boolean,
+  Balance: Double,
+  Currency: String,
+  PaymentGateway: PaymentGateway
```

Comment:
when duplicating a payment method, we now read the gateway from the zuora repsponse so that we maintain the old one in the new account regardless of whether it's New SCA or Old token.